src/theme/shared/opacity.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const opacity = {
+  20: '0.2',
+  40: '0.4',
+  60: '0.6',
+  80: '0.8',
```

Comment:
do we need `100: 1` as well? I think this could be useful if you need opacity on mobile but not on desktop.

Reply:
nice suggestion, I will add here and sync in the figma and figma-tokens. Thanks! 🥦